<launch>

  <node name="imugpspublisher4SVY21" pkg="epochmini" type="imugpspublisher4SVY21.py" />

  <node name="nofilter_pose_publisher2" pkg="epochmini" type="nofilter_pose_publisher2" />

  <node pkg="tf" type="static_transform_publisher" name="tf_maptoworld" args="0 0 0 0 0 0 map odom 30" />

 <!--  <node name="neato_laser_publisher" pkg="xv_11_laser_driver" type="neato_laser_publisher">
		<param name="port" value="/dev/ttyACM0" type="str" />
		<param name="firmware_version" value="2" type="int" />
	</node> -->

  <node name="lms111_publisher" pkg="lms1xx" type="LMS1xx_node" />

  <!-- <node pkg="tf" type="static_transform_publisher" name="tf_basetolaser" args="0 0 0 0 0 0 base_link neato_laser 30"/> -->

  <node pkg="tf" type="static_transform_publisher" name="tf_basetolaser" args="0 0 0 0 0 0 base_link laser 100" />

  <node name="local_mapping" pkg="local_map" type="local_map_out" />

  <node name="occupancytransform" pkg="epochmini" type="occupancytransform" />

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find epochmini)/navigation_params/test/costmap_common_params2.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find epochmini)/navigation_params/test/costmap_common_params2.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find epochmini)/navigation_params/test/local_costmap_params2.yaml" command="load" />
    <rosparam file="$(find epochmini)/navigation_params/test/global_costmap_params2.yaml" command="load" />
    <rosparam file="$(find epochmini)/navigation_params/test/base_local_planner_params2.yaml" command="load" />
  </node>

  <node pkg="epochmini" type="goaltracker" name="goaltracker" />

  <!-- <node name="computerVision" pkg="epochmini" type="colourtracker.py" /> -->

  <node pkg="epochmini" type="pathreader" name="pathreader" />

  <node pkg="epochmini" type="pointandgo" name="pointandgo" /> 

  <node name="rosserial_arduino" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM1" />

</launch>
